apiary.apib

```diff
@@ -1064,6 +1064,24 @@ Changing project limits is allowed only for a super admin.
 
     [Project][]
 
+## Remove Project Limit [/manage/projects/{project_id}/limits/{limit_name}]
+
+Remove project limit is allowed only for a super admin.
++ Parameters
+    + project_id (required, int) - Project ID
+    + limit_name (required, string) - Limit name
+
+### Remove project limit [DELETE]
+
++ Request (application/json)
+    + Headers
+
+            X-KBC-ManageApiToken: your_token
+
++ Response 204
+
+    [Project][]
```

Comment:
Mám podezření, že tady ten project být nemá. U jiných HTTP 204 jsem ho nenašel. Nahoře je, protože je to 200 IMHO

Reply:
![image](https://user-images.githubusercontent.com/642928/59031722-cbdee380-8864-11e9-95de-3c1a63c86512.png)

Reply:
Jo urcte to tam nema byt. Zmazem to diky

Reply:
Uz by to malo byt asi ok.